### DISCUSSION TITLE
Can you add my website link to this website?

### DISCUSSION THREAD

**Discussion by OpenSoftware-World** [ardatdev/artadosearch-source-v0.1#28] [General]
Web site Link: https://linuxuserslinuxmint.github.io

Hello, I am LinuxUsersLinuxMint and I contributed to the "Artado-search" website. And I want this website to have a link to my website or in the developers section etc. I wonder if you could add a link to my website to "Artadosearch".

@ardatdev  @Arnolxu

**Comment by cinaryilmaz** [comment]
Artado Search is a FOSS (Free & Open Source Software) project and such projects' contributors shouldn't expect any rewards for their contribution except being listed as a [contributor](https://github.com/Artado-Project/artadosearch/graphs/contributors). We appreciate your contribution but no, we can't promote your personal website in Artado.

**Comment by ardatdev** [comment]
Hi, right now we don't have that section. But we value every contribution and we are grateful for your work. Maybe we can give you and other outside contributors a role in our forum or Discord server.

**Comment by OpenSoftware-World** [comment]
> Merhaba, şu anda o bölümümüz bulunmamaktadır. Ancak her katkıya değer veriyoruz ve çalışmalarınız için minnettarız. Belki size ve diğer dışarıdan katkıda bulunanlara forumumuzda veya Discord sunucumuzda bir rol verebiliriz.



> Hi, right now we don't have that section. But we value every contribution and we are grateful for your work. Maybe we can give you and other outside contributors a role in our forum or Discord server.

Yes, I would love to come to your discord server, but your server requires phone verification. And there is no SIM card in my phone. I would appreciate it if you could remove phone verification for me. @ardatdev

**Reply by ardatdev** [reply]
We have fixed that. You can come to server now.
